software/msisensor/msi/main.nf

```diff
@@ -0,0 +1,41 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process MSISENSOR_MSI {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
+
+    conda (params.enable_conda ? "bioconda::msisensor=0.5" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/msisensor:0.5--hb3646a4_2"
+    } else {
+        container "quay.io/biocontainers/msisensor:0.5--hb3646a4_2"
+    }
+
+    input:
+    tuple val(meta), path(normal_bam), path(normal_bai), path(tumor_bam), path(tumor_bai), val(metascan), path(homopolymers)
```

Comment:
maybe this should be split up into separate channels? I'd love to hear a second opinion on that

Reply:
Yeah ... I discusssed this on slack a little with @drpatelh ... he suggested the single long tuple was best.

Reply:
I'll resolve for now, but can change if necessary ...